Framework/PythonInterface/plugins/algorithms/HB2AReduce.py

```diff
@@ -341,18 +345,41 @@ def get_vanadium(self, detector_mask, m1, colltrans, exp, indir):
             # m1 = 0 -> Ge 115, 1.54A
             # m1 = 9.45 -> Ge 113, 2.41A
             # colltrans is the collimator position, whether in or out of the beam
-            # colltrans = 0 -> IN
-            # colltrans = +/-80 -> OUT
-            vcorr_filename = 'HB2A_{}__Ge_{}_{}_vcorr.txt'.format(
-                exp, 115 if np.isclose(m1, 0, atol=0.1) else 113,
-                "IN" if np.isclose(colltrans, 0, atol=0.1) else "OUT")
+            if self.check_date():
+                # colltrans = 0 -> OUT
+                # colltrans = +/-80 -> IN
+                vcorr_filename = 'HB2A_{}__Ge_{}_{}_vcorr.txt'.format(
+                    exp, 115 if np.isclose(m1, 0, atol=0.1) else 113,
+                    "OUT" if np.isclose(colltrans, 0, atol=0.1) else "IN")
+            else:
+                # colltrans = +/-80 -> OUT
+                # colltrans = 0 -> IN
+                vcorr_filename = 'HB2A_{}__Ge_{}_{}_vcorr.txt'.format(
+                    exp, 115 if np.isclose(m1, 0, atol=0.1) else 113,
+                    "IN" if np.isclose(colltrans, 0, atol=0.1) else "OUT")
+
         vcorr_filename = os.path.join(indir, vcorr_filename)
         logger.notice("Using vcorr file: {}".format(vcorr_filename))
         if not os.path.isfile(vcorr_filename):
             raise RuntimeError("Vanadium file {} does not exist".format(vcorr_filename))
 
         return None, None, np.genfromtxt(vcorr_filename)[detector_mask]
 
+    def check_date(self):
```

Comment:
I think `path.getctime` is a great fallback position, but it would be better if you could use the contents of the file. Can you share a snippet from a file to see if there is a timestamp in it?